unit_test/test_validate_plugin.py

```diff
@@ -735,6 +736,54 @@ def test_supported_version_validator_sup_vers_empty(self):
         result = validate(directory_to_test, file_to_test, False, True, [SupportedVersionValidator()])
         self.assertEqual(result, 1)
 
+    def test_good_name_validator(self):
+        directory_to_test = "plugin_examples/name_tests"
+        file_to_test = "good_name.spec.yaml"
```

Comment:
Kindly please move that to const variables above 😄

Reply:
Let me know your thoughts.